src/views/advanced-search/results.html

```diff
@@ -231,7 +231,7 @@
       </span>
       <input class='govuk-input govuk-!-width-full' id='sicCodes' name='sicCodes' type='text' value='{{ advancedSearchParams.sicCodes }}'>
 
-      <p class='padded-top'> <a href='http://resources.companieshouse.gov.uk/sic/' target='_blank' class='govuk-link accordion-label'>Browse SIC codes (opens in new tab)</a></p>
+      <p class='padded-top'> <a href='http://resources.companieshouse.gov.uk/sic/' target='_blank' class='govuk-link accordion-label' data-event-id="advanced-search-results-page-sic-link">Browse SIC codes (opens in new tab)</a></p>
```

Comment:
To fix the security issue raised in the PR, can `rel="noopener"` be added to the element please.